payment.js

```diff
@@ -0,0 +1,120 @@
+const {MOLLIE_KEY} = process.env;
+const mollie_client = require('@mollie/api-client')({ apiKey: MOLLIE_KEY });
+
+const {createDB} = require('./db');
+const payment_database = createDB('mollie_db')
+
+// invoice: object with invoice data (straight from moneybird)
+// payment: object with payment data (straight from mollie)
+// invoice_payment: object used with the local DB, essentially a join table: {_id: invoice.id, payment_id: payment.id}
+// TODO: this currently only allows for one payment per invoice.
+
+const createPaymentData = (invoice) => new Promise(async (resolve, reject) => {
+
+    try {
+
+        const payment = await createPaymentForInvoice(invoice);
+        const invoice_payment = {_id: invoice.id, payment_id: payment.id};
+        await payment_database.insert(invoice_payment, (err, docs) => {
+            if (err) {
+                reject(err);
+            } 
+            resolve(payment);
+        });
+
+    } catch (err) {
+        reject(err);
+    }
+});
+
+
+
+const createPaymentForInvoice = (invoice) => new Promise ( async (resolve, reject) => {
+    let value;
+    // if the string only has a single character after the dot, add a zero
+    // e.g. 10.0 -> 10.00, 12.5 -> 12.50
+    if (invoice.total_unpaid.match(/\..$/)) {
+        value = `${invoice.total_unpaid}0`;
+    } else {
+        value = invoice.total_unpaid;
+    }
+
+    if (value == '0.00' && MOLLIE_KEY.startsWith('test_')) {
```

Comment:
don't write test variations in production code. if we cannot always connect to real Mollie, we should write a simple stub that implement the mollie API

Reply:
This was just convenient for testing, removed